packages/patternlab/styleguide/source/_patterns/05-pages/marketing-campaign.json

```
@@ -8,7 +8,28 @@
         "level": "1"
       },
       "textOverlay": "c-gray-lightest",
-      "bgImage": "../../assets/images/placeholder/1920x1080-sign.jpg",
+      "bgImage": {
+        "path": "@atoms/09-media/picture-element.twig",
+        "data": {
+          "picture": {
+            "defaultImage": "../../assets/images/placeholder/1600x600.png",
+            "srcSets": [
```

```suggestion
            "srcSets": [
```

the order of these images and breakpoints should be reversed

---

done